src/groupeddataframe/grouping.jl

```diff
@@ -315,26 +315,30 @@ function Base.map(f::Any, gd::GroupedDataFrame)
 end
 
 """
-    combine(gd::GroupedDataFrame, cols => f...)
+    combine(gd::GroupedDataFrame, (cols => f)...)
+    combine(gd::GroupedDataFrame, [cols1 => f1, cols2 => f2]...)
```

Comment:
This is a bit confusing (i.e. I am starting to read this PR and it is not clear for me what is the API). Typically we use a type signature. Do you mean here `::AbstractVector{<:Pair{Symbol, Callable}}`? Can multiple vectors be passed and can this vector be mixed with non vector `cols=>f` syntax? If for all question the answer is yes maybe write `combine(gd::GroupedDataFrame, args...)` and later explain what each entry of `args` can be?

Reply:
I can see below that we allow mixing, so probably the `args` approach would be cleanest.

Reply:
I think `cols` can be either a `Symbol` or a `Tuple` of `Symbols`. Yes `f` should be `Callable`. But I thought Julia was no longer using the word `Callable`.

Reply:
`Base.Callable` is defined still. But as I have written later probably it is best to write `agrs` and explain later the details.

Reply:
We can add `args...`, but having a relatively readable list of common cases sounds useful to me.